src/error.rs

```diff
@@ -1,3 +1,5 @@
+use ring::error::KeyRejected;
+use ring::error::Unspecified;
```

Comment:
These are each only used once, so prefer to just fully specify them in the place each are used, like the `http` types.